### DISCUSSION TITLE
There are a lot more failed MEV txs lately.

### DISCUSSION THREAD

**Discussion by obadiaa** [flashbots/mev-explore-public#13] [General]
![Screenshot 2021-02-22 at 18 29 17](https://user-images.githubusercontent.com/19510814/108787861-68737880-7555-11eb-8981-dc6fc8af035e.png)
(see the increase in red)

I wonder why? It is just because there are more bots online?

h/t Surya

**Comment by srya-b** [comment]
If the "more bots" explanation is true I would expect one of:
* There was some event that caused a spike in the number of bots. If so, what happened during that time? What is that event?
* There would be some sort of gradual increase in the number of bots over time rather than a sudden jump that causes the sudden jump in failed MEV txs.
